example/parsec/src/parsec.c

```diff
@@ -151,6 +183,24 @@ Parsec(S, String) identifier(void) {
   return P.label("identifier", p);
 }
 
+// -----------------------------------------------------------------------
+typedef_Fn(Token(S), UnParser(S, None));
+fn(fail_on_Fn, Token(S), UnParserArgs(S, None)) {
+  DO_WITH(c) {
+    __auto_type p = trait(ParsecDeriv(S)).anySingleBut(c);
+    for (;;) {
+      SCAN(p);
+    }
+    RETURN((None){0});
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Skipping configuration 'CPARSEC_STREAM_TYPE' since the value of 'CPARSEC_STREAM_TYPE' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly.](https://app.codacy.com/manual/mori0091/cparsec3/pullRequest?prid=5138570)

Reply:
CPARSEC_STREAM_TYPE is already defined, and it is a type-name alias therefore it shall not be a value.